.drone-1.0.yml

```diff
@@ -0,0 +1,55 @@
+
+kind: pipeline
+name: default
+type: kubernetes
+steps:
+  - name: install
+    image: node:12
+    environment:
+      TASKFLOW_POSTGRES_HOST: database
+      TASKFLOW_POSTGRES_USER: taskflow-test
+      TASKFLOW_POSTGRES_PASSWORD: test-password
```

Comment:
What is going on in `npm ci` that needs database credentials?!

Reply:
Nothing is the answer, they're left over from when this step did install _and_ test.